File: samples/sandbox/app/src/main/java/com/scichart/docsandbox/examples/java/axis3DAPIs/Axis3DLabelsLabelsConfiguration.java
package com.scichart.docsandbox.examples.java.axis3DAPIs;

import androidx.annotation.NonNull;

import com.scichart.charting3d.visuals.SciChartSurface3D;
import com.scichart.charting3d.visuals.axes.AxisPlaneDrawLabelsMode;
import com.scichart.docsandbox.core.ExampleDefinition;
import com.scichart.docsandbox.examples.base.SingleChart3DFragment;

@ExampleDefinition()
public class Axis3DLabelsLabelsConfiguration extends SingleChart3DFragment {
    @Override
    protected void initExample(@NonNull SciChartSurface3D surface) { }

    void axisCubeLabelsConfiguration(@NonNull SciChartSurface3D surface) {
        // <AxisCubeLabelsConfiguration>
        surface.setXyAxisPlaneDrawLabelsMode(AxisPlaneDrawLabelsMode.AxisPlaneDrawLabelsBoth);
        surface.setZyAxisPlaneDrawLabelsMode(AxisPlaneDrawLabelsMode.AxisPlaneDrawLabelsLocalX);
        surface.setZxAxisPlaneDrawLabelsMode(AxisPlaneDrawLabelsMode.AxisPlaneDrawLabelsHidden);
        // </AxisCubeLabelsConfiguration>
    }
}

File: samples/sandbox/app/src/main/java/com/scichart/docsandbox/examples/kotlin/axis3DAPIs/Axis3DLabelsLabelsConfiguration.kt
package com.scichart.docsandbox.examples.kotlin.axis3DAPIs

import com.scichart.charting3d.visuals.SciChartSurface3D
import com.scichart.charting3d.visuals.axes.AxisPlaneDrawLabelsMode
import com.scichart.docsandbox.core.ExampleDefinition
import com.scichart.docsandbox.examples.base.SingleChart3DFragment

@ExampleDefinition()
class Axis3DLabelsLabelsConfiguration : SingleChart3DFragment() {
    override fun initExample(surface: SciChartSurface3D) {}

    fun axisCubeLabelsConfiguration(surface: SciChartSurface3D) {
        // <AxisCubeLabelsConfiguration>
        surface.xyAxisPlaneDrawLabelsMode = AxisPlaneDrawLabelsMode.AxisPlaneDrawLabelsBoth
        surface.zyAxisPlaneDrawLabelsMode = AxisPlaneDrawLabelsMode.AxisPlaneDrawLabelsLocalX
        surface.zxAxisPlaneDrawLabelsMode = AxisPlaneDrawLabelsMode.AxisPlaneDrawLabelsHidden
        // </AxisCubeLabelsConfiguration>
    }
}
